### DISCUSSION TITLE
Directional light shadows disappear when camera is rotated

### DISCUSSION THREAD

**Discussion by GitGhillie** [bevyengine/bevy#8571] [Q&A]
Hi all,

I've loaded a single gltf, added a directional light and my player.
Everything looks fine when the camera is horizontal:
<img src="https://user-images.githubusercontent.com/23659539/236801601-59700c50-1fb3-4795-8872-4eca5c24711a.png" width="500">

Slightly looking down some of the shadows start to disappear:

<img src="https://user-images.githubusercontent.com/23659539/236801705-635e536f-27d2-4688-9420-65835659268b.png" width="500">

Any idea what could be causing this?
I've got DirectionalLightShadowMap size set to 4096 and playing around with the CascadeShadowConfigBuilder did not seem to change anything other than the quality of the shadows.

Camera3dBundle settings:
```
Camera3dBundle {
                transform: Transform::from_xyz(0.0, 1.8, 0.0),
                projection: Projection::Perspective(PerspectiveProjection { fov: 1.0, aspect_ratio: 1.859, near: 0.1, far: 1000.0 }),
                ..default() },
```

**Comment by superdump** [comment]
I wonder if this is a problem with normals.

**Comment by GitGhillie** [comment]
Fixed by #8645
